README.md

```diff
@@ -261,6 +261,38 @@ NSNumber *shouldUseLiterals = [NSNumber numberWithBool:YES];
 NSNumber *buildingZIPCode = [NSNumber numberWithInteger:10018];
 ```
 
+When writing a multiline literal or enumeration, leave a trailing comma on the last item. When entries are added or removed, this eliminates the need to touch unrelated lines. This eliminates irrelevant git history from the unrelated lines of code, and it makes for cleaner diffs; only the truly relevant lines of code must be changed.
```

Comment:
> Existing `enums` and arrays in other parts of the guide should also be updated.

Done in https://github.com/cdzombak/objective-c-style-guide/commit/4ebd3ce2733cb1f77a335c5bc680d47fabcee9ca .